src/AppBundle/Helper/EditCounterHelper.php

```diff
@@ -143,6 +140,7 @@ public function getRevisionCounts($userId)
             3
         );
 
+        $this->cacheSave($cacheKey, $revisionCounts, 'P1D');
```

Comment:
This I don't think should be cached at all. Revision counts change frequently and people will probably complain that the numbers are off